classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+  attr_reader :students
+
+  def add_student(student)
```

Comment:
- [ ] Great job so far. please kindly set a class for student and ensure When setting the classroom for a student it also adds it to the classrooms' students
- [ ] also please add to it a method that serves as the 'belongs-to' side (a student belongs to a classroom)`

eg
 ```ruby
class Student < Person
  attr_reader :classroom

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```

Reply:
Thank you for your feedback